package io.fiap.erp.exception;

public class UsuarioNaoExisteException extends RuntimeException {

    public UsuarioNaoExisteException(){}

    public UsuarioNaoExisteException(String message){}

    public UsuarioNaoExisteException(String message, Throwable cause){}

    public UsuarioNaoExisteException(Throwable cause) {}

}
